src/components/MainApp/MainApp.js

```diff
@@ -46,6 +47,9 @@ function MainApp(props) {
       <div className="app-main">
 
         <Switch>
+        <Route path="/u/">
```

Comment:
You may want to rebase off `main`. I don't think it'll be too hard to resolve conflicts - this part will just need to be here instead:

https://github.com/0x213F/jukebox-radio-react/blob/main/src/components/MainApp/MainApp.js#L20